pkg/network/multus_admission_controller.go

```diff
@@ -102,6 +110,30 @@ func renderMultusAdmissonControllerConfig(manifestDir string, externalControlPla
 		data.Data["ClusterIDLabel"] = hypershift.ClusterIDLabel
 		data.Data["ClusterID"] = bootstrapResult.Infra.HostedControlPlane.ClusterID
 		data.Data["HCPNodeSelector"] = bootstrapResult.Infra.HostedControlPlane.NodeSelector
+		data.Data["PriorityClass"] = bootstrapResult.Infra.HostedControlPlane.PriorityClass
+
+		// Preserve any existing multus container resource requests which may have been modified by an external source
```

Comment:
I think this blocks us from changing the requests values for hypershift in future openshift versions.
On upgrade the old deployment exists and we will just copy the previous values.
Could you link link any other place in which hypershift does that?

Reply:
@kyrtapz Thanks for taking a look!

> Could you link any other example where hypershift does that?

I believe a similar approach was taken in a couple other places:
* https://github.com/openshift/hypershift/pull/1082
* https://github.com/openshift/hypershift/pull/3120
* https://github.com/openshift/hypershift/pull/3864

Reply:
Thanks! I've talked to @csrwng and this is the expected behavior in HyperShift.